### PR TITLE
Fix wrong profile information for TV Maps app

### PR DESCRIPTION
**Pull request by sgchoi5**
(M) => (T)

Added missing path : /Xamarin.Forms/

### CODE REVIEW

**Review by nawab-samsung — APPROVED** [review]
@sgchoi5 Thanks.

**Comment by sgchoi5** [comment]
Fixed more wrong path: https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/WeatherWatch
https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/**Xamarin.Forms**/WeatherWatch

File: docs/application/dotnet/samples/watch/overview.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ for TD:
 			<td>
                         <p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/Xamarin.Forms/WeatherWatch" target="_blank"><strong>(W) WeatherWatch</strong></a></p>
 			<p>This sample application demonstrates how to create a watch face application with weather information for Tizen wearable devices. It also demonstrates how to use watch face API using <a href="https://samsung.github.io/Tizen.CircularUI/api/index.html" target="_blank">Tizen.Wearable.CircularUI</a>.<br>
-			You can get detailed information from <a href="https://github.com/Samsung/Tizen-CSharp-Samples/blob/master/Wearable/WeatherWatch" target="_blank"><strong>readme file</strong> of WeatherWatch</a>.</p>
+			You can get detailed information from <a href="https://github.com/Samsung/Tizen-CSharp-Samples/blob/master/Wearable/Xamarin.Forms/WeatherWatch" target="_blank"><strong>readme file</strong> of WeatherWatch</a>.</p>
```
Comment (nawab-samsung) [inline docs/application/dotnet/samples/watch/overview.md:112]:
```suggestion 
                        You can get detailed information from <a href="https://github.com/Samsung/Tizen-CSharp-Samples/blob/master/Wearable/Xamarin.Forms/WeatherWatch/README.md" target="_blank"><strong>the readme file</strong></a> of WeatherWatch.</p>

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
@sgchoi5 How about this suggestion to give the path to readme file?

File: docs/application/dotnet/samples/watch/overview.md (hunk 1 above)
Comment (sgchoi5) [reply docs/application/dotnet/samples/watch/overview.md]: Updated!

**Review by nawab-samsung — APPROVED** [review]
@sgchoi5 thanks a lot for applying my suggestion.

**Pull request merged**